cli/admin/clusters.go

```diff
@@ -0,0 +1,62 @@
+//
```

Comment:
since the cluster create/delete/modify is used mainly for replication - it might be good to create a replication command group and put these underneath.